### PR TITLE
Turn exercise 013 into practice

### PR DESCRIPTION
**Pull request by divyankachaudhari**
For #2198.

`dune build --root .` works. 

Addresses all potential issues pointed out at #2179.

### CODE REVIEW

File: practice/013/ex.ml
Comment (cuihtlauac) [inline practice/013/ex.ml:1]: Edit this file to minimize the diff against `011/ex.ml`

File: practice/013/answer/impl.ml
Comment (cuihtlauac) [inline practice/013/answer/impl.ml:1]: Edit this file to minimize the diff against `011/answer/impl.ml`

**Review by cuihtlauac — COMMENTED** [review]
Nice @divyankachaudhari. could you deal with 010 too, it's the same topic

File: practice/013/ex.ml
Comment (divyankachaudhari) [reply practice/013/ex.ml:1]: Done.

File: practice/013/answer/impl.ml
Comment (divyankachaudhari) [reply practice/013/answer/impl.ml:1]: Done

**Comment by cuihtlauac** [comment]
`dune test --root .` is failing on my side. Is it on yours too?

**Comment by divyankachaudhari** [comment]
> `dune test --root .` is failing on my side. Is it on yours too?

@cuihtlauac Shouldn't it be `dune build --root .`?

**Comment by divyankachaudhari** [comment]
Please continue on #2204, I'm closing this PR.

**Pull request closed without merging**